### ISSUE TITLE
StackingEstimator truncates parameters

### ISSUE THREAD

**Issue by abhishekvij** [EpistasisLab/tpot#789]
When an algorithm is added to the pipeline under StackingEstimator the named_steps  component has truncated set of the parameters. Let me given an example to elaborate.

## Process to reproduce the issue

testp = make_pipeline(
   StackingEstimator(estimator= GradientBoostingRegressor(alpha=0.9, learning_rate=0.1, loss="quantile", max_depth=1, max_features=0.9500000000000001, min_samples_leaf=18, min_samples_split=17, n_estimators=100, subsample=0.6500000000000001)),
)

## Expected result

 GradientBoostingRegressor(alpha=0.9, criterion='friedman_mse', init=None,
              learning_rate=0.1, loss='quantile', max_depth=1,
              max_features=0.9500000000000001, max_leaf_nodes=None,
              min_impurity_decrease=0.0, min_impurity_split=None,
              min_samples_leaf=18, min_samples_split=17,
              min_weight_fraction_leaf=0.0, n_estimators=100,
              presort='auto', random_state=None,
              subsample=0.6500000000000001, verbose=0, warm_start=False))

## Current result

('stackingestimator',
 StackingEstimator(estimator=GradientBoostingRegressor(alpha=0.9, criterion='friedman_mse', init=None,
              learning_rate=0.1, loss='quantile', max_depth=1,
              max_features=0.9500000000000001, max_leaf_nodes=None,
              min_impurity_decrease=0.0, min_impurity_split=None,
              min_sampl...t='auto', random_state=None,
              subsample=0.6500000000000001, verbose=0, warm_start=False)))

## Further explanation of issue

As you can see in the current result the pipeline parameters are truncated at "min_sampl...t='auto'", this only happens when stacking estimators are used and a lot of parameters are passed to the algo.

This doesn't hinder with the making of the pipeline or the fit method but it's a problem when you are trying to analyze the individual estimators and what is going in each one of these estimators.

**Comment by weixuanfu** [comment]
You may print out all parameters via [`get_params`](http://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html#sklearn.base.BaseEstimator.get_params), like `print(testp.get_params())`.

**Comment by abhishekvij** [comment]
Thanks a lot @weixuanfu .

Just a note, if someone wanders here to find a solution. you can get the estimator parameters as well as the entire estimator using.

temp = testp.get_params()
temp['stackingestimator__estimator']

This gives the following output

GradientBoostingRegressor(alpha=0.9, criterion='friedman_mse', init=None,
             learning_rate=0.1, loss='quantile', max_depth=1,
             max_features=0.9500000000000001, max_leaf_nodes=None,
             min_impurity_decrease=0.0, min_impurity_split=None,
             min_samples_leaf=18, min_samples_split=17,
             min_weight_fraction_leaf=0.0, n_estimators=100,
             presort='auto', random_state=None,
             subsample=0.6500000000000001, verbose=0, warm_start=False)
